pygotham/talks/forms.py

```diff
@@ -23,7 +23,12 @@ class TalkSubmissionForm(ModelForm):
 
     class Meta:
         model = Talk
-        exclude = ('status',)
+        exclude = ('abstract',
+                   'objectives',
+                   'outline',
+                   'status',
+                   'target_audience',
+                   'type')
```

Comment:
Just to compare:

```
exclude = (
    'abstract',
    'objectives',
)
```
vs

```
exclude = ('abstract',
           'objectives')
```
The former will have much simpler diffs when you add and remove new items.